src/ambianic/pipeline/ai/fall_detect.py

```diff
@@ -88,7 +90,17 @@ def calculate_angle(self, p):
         theta = abs(theta1-theta2)
         return theta
 
-
+    def calculate_angle_with_yaxis(self, p):
+        
+        x3, y3 = p[0][0], p[0][1]
+        x4, y4 = p[1][0], p[1][1]
+        
+        theta1 = 90
```

Comment:
@bhavikapanara have you verified which quadrant is the 90' vector? I've noticed that the 90' rotation with PIL.transpose() places the image in a quadrant below the X-axis, as opposed to above the X-axis as I expected. This probably has to do with the reverse Y-axis coordinates in image pixels.

Can you please visualize test_case_5 in a colab so we can be sure the vectors are correct.

It may be safer to use a vector y_axis=((0, image.height), (0, 0)) and apply the existing method:
```
calculate_angle(body_line, y-axis).
```

Think about it and let me know your thoughts.

Reply:
Yes, @ivelin 

You are correct...I have updated the code.